editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/AdvancedEndpointPropertiesSection.java

```diff
@@ -340,8 +305,11 @@ public IStatus validate(Object value) {
 				new UnsupportedParameterPropertyUICreatorForAdvanced(dbc, modelMap, eip, selectedEP, p, page, getWidgetFactory()).create();
             // CLASS BASED PROPERTIES - REF OR CLASSNAMES AS STRINGS
             } else {
-				new ClassBasedParameterPropertyUICreatorForAdvanced(dbc, modelMap, eip, selectedEP, prop, page, getWidgetFactory()).create();
+            	// for now we disable handling for this type - FUSETOOLS-2188
+//            	new ClassBasedParameterPropertyUICreatorForAdvanced(dbc, modelMap, eip, selectedEP, prop, page, getWidgetFactory()).create();
```

Comment:
the class will still be available in history. Not sure that we will be able to reuse a big part of it anyway and we don't know when we will do it

I won't block the merge for that but I really think that it is a bad idea to keep it.